apax/cli/templates/train_config_full.yaml

```diff
@@ -38,14 +38,16 @@ data:
   energy_unit: eV
 
 model:
-  n_basis: 7
+  basis:
+    name: gaussian
+    n_basis: 7
+    r_max: 6.0
+    r_min: 0.5
```

Comment:
This will break existing API and all existing models, won't it?

Reply:
Yes, but the recent dataset update did the same. Old models aren't strictly "broken", one just needs to adapt the config file.